runners/flink/src/main/java/org/apache/beam/runners/flink/LyftFlinkStreamingPortableTranslations.java

```diff
@@ -193,33 +225,63 @@ public boolean isEndOfStream(WindowedValue<byte[]> nextElement) {
   }
 
   // TODO: translation assumes byte[] values, does not support keys and headers
-  private void translateKafkaSink(
+  @VisibleForTesting
+  void translateKafkaSink(
       String id, RunnerApi.Pipeline pipeline, StreamingTranslationContext context) {
     RunnerApi.PTransform pTransform = pipeline.getComponents().getTransformsOrThrow(id);
     final String topic;
-    final Properties properties = new Properties();
+    final String bootstrapServers;
+
     ObjectMapper mapper = new ObjectMapper();
+    FlinkKafkaProducer011<WindowedValue<byte[]>> producer;
+    LyftKafkaProducerBuilder<WindowedValue<byte[]>> producerBuilder =
+        new LyftKafkaProducerBuilder<>();
     try {
       Map<String, Object> params =
           mapper.readValue(pTransform.getSpec().getPayload().toByteArray(), Map.class);
 
       Preconditions.checkNotNull(topic = (String) params.get("topic"), "'topic' needs to be set");
 
-      Map<?, ?> consumerProps = (Map) params.get("properties");
-      Preconditions.checkNotNull(consumerProps, "'properties' need to be set");
-      properties.putAll(consumerProps);
+      Map<?, ?> producerProps = (Map) params.get("properties");
+      Preconditions.checkNotNull(producerProps, "'properties' need to be set");
+      logger.info(
+          "Configuring kafka producer for topic {} with properties {}", topic, producerProps);
+
+      Preconditions.checkNotNull(
+          bootstrapServers = (String) producerProps.remove("bootstrap.servers"),
+          "'bootstrap.servers' needs to be set");
+
+      final String splits[] = bootstrapServers.split(":", 2);
+      final String bootstrapServer = splits[0];
```

Comment:
Again, it seems like this would not support a list of bootstrap servers where the first server uses the standard port. I think we should split by `,` and then by `:`.